sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TransmitterFactory.cs

```diff
@@ -29,7 +29,7 @@ public ITransmitter Get(AzureMonitorExporterOptions azureMonitorExporterOptions)
                 {
                     if (!_transmitters.TryGetValue(key, out transmitter))
                     {
-                        transmitter = new AzureMonitorTransmitter(azureMonitorExporterOptions, new DefaultPlatform());
+                        transmitter = new AzureMonitorTransmitter(azureMonitorExporterOptions, platform ?? DefaultPlatform.Instance);
```

Comment:
`platform ?? DefaultPlatform.Instance`

Do we need this check?

Reply:
Removed